pkg/sql/crdb_internal.go

```diff
@@ -3028,9 +3028,6 @@ CREATE TABLE crdb_internal.ranges_no_leases (
 )
 `,
 	generator: func(ctx context.Context, p *planner, _ catalog.DatabaseDescriptor, _ *stop.Stopper) (virtualTableGenerator, cleanupFunc, error) {
-		if err := p.RequireAdminRole(ctx, "read crdb_internal.ranges_no_leases"); err != nil {
```

Comment:
We should make it such that if they have admin OR `ZONECONFIG` they can perform this.

root does not have `ZONECONFIG` on system tables it seems.

```
demo@127.0.0.1:26257/movr> show grants on system.jobs for root;
  database_name | schema_name | table_name | grantee | privilege_type
----------------+-------------+------------+---------+-----------------
  system        | public      | jobs       | root    | DELETE
  system        | public      | jobs       | root    | GRANT
  system        | public      | jobs       | root    | INSERT
  system        | public      | jobs       | root    | SELECT
  system        | public      | jobs       | root    | UPDATE
```

Reply:
Thanks for the early review Richard! Yes either having the admin role or `ZONECONFIG` makes more sense, I've made the changes.

Reply:
@RichardJCai I'm starting to think it may not be possible, see my comment below